### PR TITLE
Fix virt-v2v entrypoint script

### PR DESCRIPTION
**Pull request by liranr23**
Since we added OVA we had two options to execute the virt-v2v command. Now the command will be built and will be executed at the end without conditions regarding the source. This will allow us to know if we had an error and fail the pod in such case.

### CODE REVIEW

File: virt-v2v/cold/entrypoint [hunk 1]
```diff
@@ -87,22 +84,11 @@ if [ "$V2V_source" == "vCenter" ] ; then
           -io "vddk-thumbprint=$V2V_thumbprint"
       )
   fi
+  args+=(-- "$V2V_vmName")
```
Comment (ahadas) [inline virt-v2v/cold/entrypoint:87]: don't we need to append above (line 81)?

Comment (ahadas) [reply virt-v2v/cold/entrypoint:87]: I mean, for consistency...

Comment (liranr23) [reply virt-v2v/cold/entrypoint:87]: no, because line 81 is only when VDDK is available.

Comment (ahadas) [reply virt-v2v/cold/entrypoint:87]: > no, because line 81 is only when VDDK is available.

let me rephrase, wouldn't it be better to replace line 81 with:
```golang
args+=(
```

**Review by ahadas — APPROVED** [review]

**Comment by ahadas** [comment]
Much better this way!
